src/app/forms/templates/test-records/section-templates/vehicle/desk-based-default-hgv-trl-vehicle-section.template.ts

```diff
@@ -0,0 +1,77 @@
+import { AsyncValidatorNames } from '@forms/models/async-validators.enum';
+import { ValidatorNames } from '@forms/models/validators.enum';
+import { FormNode, FormNodeEditTypes, FormNodeTypes, FormNodeViewTypes, FormNodeWidth } from '@forms/services/dynamic-form.types';
+import { ReferenceDataResourceType } from '@models/reference-data.model';
+
+export const DeskBasedVehicleSectionDefaultHgvTrl: FormNode = {
+  name: 'vehicleSection',
+  label: 'Vehicle Details',
+  type: FormNodeTypes.GROUP,
+  children: [
+    {
+      name: 'vin',
+      label: 'VIN/chassis number',
+      value: '',
+      type: FormNodeTypes.CONTROL,
+      viewType: FormNodeViewTypes.HIDDEN,
+      editType: FormNodeEditTypes.HIDDEN
+    },
+    {
+      name: 'vrm',
+      label: 'VRM',
+      value: '',
+      type: FormNodeTypes.CONTROL,
+      viewType: FormNodeViewTypes.HIDDEN,
+      editType: FormNodeEditTypes.HIDDEN
+    },
+    {
+      name: 'countryOfRegistration',
+      label: 'Country Of Registration',
+      value: '',
+      editType: FormNodeEditTypes.AUTOCOMPLETE,
+      options: [],
+      referenceData: ReferenceDataResourceType.CountryOfRegistration,
+      type: FormNodeTypes.CONTROL
+    },
+    {
+      name: 'euVehicleCategory',
+      label: 'EU Vehicle Category',
+      value: '',
+      disabled: true,
+      type: FormNodeTypes.CONTROL,
+      width: FormNodeWidth.XXS
+    },
+    {
+      name: 'odometerReading',
```

Comment:
The odometer reading/units for a trailer doesn't seem right, on all other trailers it's not there and doesn't accept a value even if it's null

Reply:
Split this into two templates, one with odometer for hgv, one without for trl